pointgrey_camera_driver/cfg/PointGrey.cfg

```diff
@@ -109,8 +111,11 @@ gen.add("tilt",               int_t, 	SensorLevels.RECONFIGURE_RUNNING,     "Con
 
 gen.add("brightness",         double_t, SensorLevels.RECONFIGURE_RUNNING,     "Black level offset.",				                                             0.0, 0.0, 10.0)
 
+gen.add("sharpness",         double_t, SensorLevels.RECONFIGURE_RUNNING,     "sharpness.",				                                             1024.0, 0.0, 4095.0)
+gen.add("saturation",         double_t, SensorLevels.RECONFIGURE_RUNNING,     "saturation.",				                                              100.0, 0.0, 100.0)
```

Comment:
Thanks for the review. I made sure the range is the same as the old driver. One thing though, eventhough the auto-sharpness button in the auto-reconfigure seems to have an effect on switching sharpness from auto to manual, the sharpness slider doesn't seem to have an effect on the value. I am still investigating this.